tests/testthat/test-seq_linter.R

```diff
@@ -49,6 +49,19 @@ test_that("finds 1:length(...) expressions", {
     rex("dim(...)", anything, "Use seq_len"),
     linter
   )
+
+  expect_lint(
+    "function(x) { mutate(x, .id = 1:n()) }",
+    rex("n(...)", anything, "Use seq_len"),
```

Comment:
not ideal that the message is producing n(...) since n() doesn't take any arguments:

https://dplyr.tidyverse.org/reference/context.html

would you mind having a go at special-casing n() in the message generation so that the message is n() not n(...)?

if you're not having any luck, that's OK -- we can fix it in follow-up. the message for seq_linter needs improvement more generally anyway.

Reply:
Good catch!

Yeah, I can give it a try.

Reply:
@MichaelChirico Just pushed something to deal with `n()` in special way.

Let me know if you think it's not the best way to proceed on this.